implementations/cli/src/main/java/de/bluecolored/bluemap/cli/BlueMapCLI.java

```diff
@@ -254,6 +255,12 @@ public static void main(String[] args) {
 		try {
 			CommandLine cmd = parser.parse(BlueMapCLI.createOptions(), args, false);
 			
+			if (cmd.hasOption("l")) {
+				if (Logger.global instanceof LoggerLogger) {
+					((LoggerLogger) Logger.global).addFileHandler(cmd.getOptionValue("l"), cmd.hasOption("a"));
+				}
```

Comment:
Instead of this, can we create a new `LoggerLogger` instance with the file-handler attached and set `Logger.global` to that new instance?
I think this is cleaner than relying on `Logger.global` being an instance of `LoggerLogger` :)